README.md

```
@@ -101,11 +101,23 @@ This method needs no additional pre-requirement or dependency, just:
+If you run into the following error:
+
+> Apple could not verify “Transmission Remote GUI” ...
+
+See #1499 for more info. Alternatively, use the Homebrew method.
+
 #### Homebrew
 
-You need to have [Homebrew](https://brew.sh/) installed. Execute this command to install Transmission Remote Gui:
+You need to have [Homebrew](https://brew.sh/) installed. Execute these two commands to install Transmission Remote GUI:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)